src/main/java/solutions/siren/join/common/Bytes.java

```diff
@@ -114,4 +126,34 @@ public final static long readLong(BytesRef src) {
     return (((long) readInt(src)) << 32) | (readInt(src) & 0xFFFFFFFFL);
   }
 
+  public final static void writeBytesRef(BytesRef src, BytesRef dst) {
+    if (src == null) {
```

Comment:
ah ok I see in `readBytesRef` it reads a zero and acts accordingly. Maybe a comment would be enough.